src/web/lib/getBrazeUuid.ts

```diff
@@ -0,0 +1,22 @@
+import { joinUrl } from '@root/src/web/lib/joinUrl';
+
+export const getBrazeUuid = async (ajaxUrl: string): Promise<string> => {
+    const url = joinUrl([ajaxUrl, 'user/me']);
```

Comment:
incidentally I think the identity team were looking into whether they could remove calls to `user/me` on every page view (or at least it was on their backlog). Better let them know there's a new use case for this if you're going to merge something similar to this.

Reply:
Ah cool thanks, thats good to know :+1: We noticed that we were making a call to `/user/me` on every page for non-dcr rendered pages and it's on my radar to figure out if we could persist the data we need (the braze UUID) somewhere so we don't need to make the request every time.